jenkins-slaves/jenkins-slave-npm/Dockerfile

```diff
@@ -0,0 +1,33 @@
+#invoke npm in jenkinsfile: sh "scl enable rh-nodejs6 'npm run build'"
+FROM registry.access.redhat.com/openshift3/jenkins-slave-base-rhel7
+USER root
```

Comment:
Please remove, `openshift3/jenkins-slave-base-rhel7` is already running as root.